### ISSUE TITLE
remove logging code in ARAX_expand.py

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#2133]
@amykglen,  I wonder if this logging code should be removed, since logging is configured in `__main__.py` of the Flask application and I'm not sure we should be clearing its logging handlers:

https://github.com/RTXteam/RTX/blob/d5c210e172566aa46f7786216a52449ae37fa843/code/ARAX/ARAXQuery/ARAX_expander.py#L52-L58

and I think maybe we should change all instances of `self.logger.WHATEVER(` in this module to use TRAPI logging or `logging.debug(` or whatever.

**Comment by amykglen** [comment]
@saramsey - yes, that sounds good - I think this was added way back in the days when we used `multiprocessing` instead of `asyncio`, so it isn't relevant anymore (`logging` is only logged to when in multiprocessing mode, which we don't use anymore - I think I did that because there were some issues with having children processes write to TRAPI logs for whatever reason... fuzzy on the details). at the time I think I thought no one else was using `logging` - certainly not good to be clearing others' handlers! definitely should be removed.

on a related note - I've been meaning for a while to remove the option to use `multiprocessing` instead of `asyncio`. we wanted to keep that for a while after switching to `asyncio` just in case it wasn't working well, but now we're firmly in the `asyncio` paradigm, and the `multiprocessing` option complicates the code quite a bit.
